ci-operator/step-registry/cilium/conf/cilium-conf-commands.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+cat >> "${SHARED_DIR}/install-config.yaml" << EOF
+networking:
+  networkType: Cilium
```

Comment:
If I understood correctly this effectively disables cluster network operator by setting some unknown network type (so that we could setup network from manifests later)?

Lets make this step more generic - `ci-operator/step-registry/network/conf/disable-cno/...` and set `networkType: none` (this would allow us to reuse this step for other CNIs)

Reply:
Good idea, I will do this.